From 1c5d10c7858387de9bd7e2577a61ede3a6334352 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 9 Dec 2017 16:30:43 +0100
Subject: [PATCH 1/1] mx6: wandboard: reenable SPL build

The last merge from the u-boot-imx tree deleted a
necessary line from wandboard_defconfig by mistake.

This caused an error when running

	make wandboard_defconfig
	make SPL

Fixes: 6e6cf015e7cd Merge git://www.denx.de/git/u-boot-imx
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 configs/wandboard_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/wandboard_defconfig b/configs/wandboard_defconfig
index 4b6b9d9aaee..0e1f1f4a376 100644
--- a/configs/wandboard_defconfig
+++ b/configs/wandboard_defconfig
@@ -11,6 +11,7 @@ CONFIG_SPL_WATCHDOG_SUPPORT=y
 CONFIG_CMD_HDMIDETECT=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_BOOTCOMMAND="run findfdt; run finduuid; run distro_bootcmd"
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=arch/arm/mach-imx/spl_sd.cfg"
 # CONFIG_CONSOLE_MUX is not set
 CONFIG_SYS_CONSOLE_IS_IN_ENV=y
 CONFIG_SYS_CONSOLE_OVERWRITE_ROUTINE=y
-- 
2.11.0

